addons/default/visiosoft/advs-module/src/Http/Controller/advsController.php

```diff
@@ -2,47 +2,46 @@
 use Anomaly\Streams\Platform\Model\Advs\PurchasePurchaseEntryModel;
 use Anomaly\Streams\Platform\Model\Complaints\ComplaintsComplainTypesEntryModel;
 use Anomaly\Streams\Platform\Model\Options\OptionsAdvertisementEntryModel;
 use Anomaly\UsersModule\User\Contract\UserRepositoryInterface;
-use Visiosoft\AdvsModule\Adv\Command\appendRequestURL;
-use Visiosoft\AdvsModule\Adv\Event\ChangedStatusAd;
-use Visiosoft\AdvsModule\Adv\Event\CreatedAd;
-use Visiosoft\AdvsModule\Adv\Event\showAdPhone;
+use Illuminate\Contracts\Events\Dispatcher;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Auth;
 use Illuminate\Support\Facades\Cookie;
-use Visiosoft\AdvsModule\Option\Contract\OptionRepositoryInterface;
-use Visiosoft\LocationModule\City\CityRepository;
-use Visiosoft\ProfileModule\Adress\Contract\AdressRepositoryInterface;
-use function PMA\Util\get;
 use Visiosoft\AdvsModule\Adv\AdvModel;
+use Visiosoft\AdvsModule\Adv\Command\appendRequestURL;
+use Visiosoft\AdvsModule\Adv\Contract\AdvRepositoryInterface;
+use Visiosoft\AdvsModule\Adv\Event\ChangedStatusAd;
+use Visiosoft\AdvsModule\Adv\Event\CreatedAd;
 use Visiosoft\AdvsModule\Adv\Event\priceChange;
+use Visiosoft\AdvsModule\Adv\Event\showAdPhone;
 use Visiosoft\AdvsModule\Adv\Event\UpdateAd;
 use Visiosoft\AdvsModule\Adv\Event\viewAd;
 use Visiosoft\AdvsModule\Adv\Form\AdvFormBuilder;
-use Visiosoft\CatsModule\Category\CategoryModel;
-use Visiosoft\CommentsModule\Comment\CommentModel;
-use Visiosoft\LocationModule\City\CityModel;
+use Visiosoft\AdvsModule\Option\Contract\OptionRepositoryInterface;
 use Visiosoft\AlgoliaModule\Search\SearchModel;
 use Visiosoft\AlgoliatestModule\Http\Controller\Admin\IndexController;
+use Visiosoft\CatsModule\Category\CategoryModel;
+use Visiosoft\CatsModule\Category\Contract\CategoryRepositoryInterface;
 use Visiosoft\CloudinaryModule\Video\VideoModel;
+use Visiosoft\CommentsModule\Comment\Events\CreateNewComment;
 use Visiosoft\FavsModule\Http\Controller\FavsController;
+use Visiosoft\LocationModule\City\CityModel;
+use Visiosoft\LocationModule\City\CityRepository;
+use Visiosoft\LocationModule\Country\Contract\CountryRepositoryInterface;
 use Visiosoft\LocationModule\District\DistrictModel;
 use Visiosoft\LocationModule\Neighborhood\NeighborhoodModel;
 use Visiosoft\LocationModule\Village\VillageModel;
 use Visiosoft\PackagesModule\Http\Controller\PackageFEController;
-use Visiosoft\AdvsModule\Adv\Contract\AdvRepositoryInterface;
-use Visiosoft\CatsModule\Category\Contract\CategoryRepositoryInterface;
-use Visiosoft\LocationModule\Country\Contract\CountryRepositoryInterface;
-use Anomaly\Streams\Platform\Message\MessageBag;
 use Visiosoft\PackagesModule\Package\PackageModel;
-
-use Illuminate\Contracts\Events\Dispatcher;
+use Visiosoft\ProfileModule\Adress\Contract\AdressRepositoryInterface;
 use Visiosoft\QrcontactModule\Qr\QrModel;
 use Visiosoft\StoreModule\Ad\AdModel;
+use function PMA\Util\get;
```

Comment:
do we need? @muammertop

Reply:
I didn't add it, but I guess it's not working for us.

Reply:
php storm could have been add that. just remove that line